mercury_app/utils.py

```diff
@@ -192,3 +197,91 @@ def get_events_for_organizations(organizations, user):
         for e in event:
             e['org_name'] = organization['name']
     return event
+
+
+def create_webhook_from_view(user):
+    if not UserWebhook.objects.filter(user=user).exists():
+        token = get_auth_token(user)
+        webhook_id = create_webhook(token)
+        UserWebhook.objects.create(
+            user=user,
+            webhook_id=webhook_id,
+        )
+
+
+def create_webhook(token):
```

Comment:
If this is not a generic "webhooks creator" please call it as "create_order_webhook". If it is generic it could receive params.